2025-06-21 13:53:48,443 INFO [main]: Запуск калькулятора теплоизоляции НФС v0.1...
2025-06-21 13:53:48,444 INFO [check_dependencies]: Начата проверка наличия необходимых библиотек...
2025-06-21 13:53:48,445 INFO [check_dependencies]: Все необходимые библиотеки присутствуют.
2025-06-21 13:53:48,445 INFO [initialize_database]: Начата инициализация базы данных...
2025-06-21 13:53:48,445 INFO [sync_db_with_fixture]: Синхронизация базы данных с фикстурой...
2025-06-21 13:53:48,510 INFO [sync_db_with_fixture]: Соединение с базой данных установлено
2025-06-21 13:53:48,511 INFO [load_fixture_data]: Загрузка даных из фикстуры: assets/material.json
2025-06-21 13:53:48,511 ERROR [load_fixture_data]: Файл фикстуры не найден: assets/material.json
2025-06-21 13:53:48,511 ERROR [sync_db_with_fixture]: Неизвестная ошибка при синхронизации: [Errno 2] No such file or directory: 'assets/material.json'
2025-06-21 13:53:48,511 ERROR [initialize_database]: Ошибка инициализации БД: [Errno 2] No such file or directory: 'assets/material.json'
Traceback (most recent call last):
  File "C:\PetProject\Calculations\ventilated_fasade\main.py", line 68, in initialize_database
    sync_db_with_fixture(FIXTURE_PATH, DB_URL)
  File "C:\PetProject\Calculations\ventilated_fasade\data\sync.py", line 39, in sync_db_with_fixture
    data = load_fixture_data(fixture_path)
  File "C:\PetProject\Calculations\ventilated_fasade\data\sync.py", line 16, in load_fixture_data
    with open(path, 'r', encoding='utf-8') as file:
FileNotFoundError: [Errno 2] No such file or directory: 'assets/material.json'
2025-06-21 13:53:52,962 WARNING [main]: Завершение работы из-за ошибки инициализации БД.
2025-06-21 13:55:02,013 INFO [main]: Запуск калькулятора теплоизоляции НФС v0.1...
2025-06-21 13:55:02,014 INFO [check_dependencies]: Начата проверка наличия необходимых библиотек...
2025-06-21 13:55:02,014 INFO [check_dependencies]: Все необходимые библиотеки присутствуют.
2025-06-21 13:55:02,015 INFO [initialize_database]: Начата инициализация базы данных...
2025-06-21 13:55:02,015 INFO [sync_db_with_fixture]: Синхронизация базы данных с фикстурой...
2025-06-21 13:55:02,042 INFO [sync_db_with_fixture]: Соединение с базой данных установлено
2025-06-21 13:55:02,042 INFO [load_fixture_data]: Загрузка даных из фикстуры: assets/material.json
2025-06-21 13:55:02,042 ERROR [load_fixture_data]: Файл фикстуры не найден: assets/material.json
2025-06-21 13:55:02,043 ERROR [sync_db_with_fixture]: Неизвестная ошибка при синхронизации: [Errno 2] No such file or directory: 'assets/material.json'
2025-06-21 13:55:02,043 ERROR [initialize_database]: Ошибка инициализации БД: [Errno 2] No such file or directory: 'assets/material.json'
Traceback (most recent call last):
  File "C:\PetProject\Calculations\ventilated_fasade\main.py", line 68, in initialize_database
    sync_db_with_fixture(FIXTURE_PATH, DB_URL)
  File "C:\PetProject\Calculations\ventilated_fasade\data\sync.py", line 39, in sync_db_with_fixture
    data = load_fixture_data(fixture_path)
  File "C:\PetProject\Calculations\ventilated_fasade\data\sync.py", line 16, in load_fixture_data
    with open(path, 'r', encoding='utf-8') as file:
FileNotFoundError: [Errno 2] No such file or directory: 'assets/material.json'
2025-06-21 13:55:08,113 WARNING [main]: Завершение работы из-за ошибки инициализации БД.
2025-06-21 13:55:48,125 INFO [main]: Запуск калькулятора теплоизоляции НФС v0.1...
2025-06-21 13:55:48,126 INFO [check_dependencies]: Начата проверка наличия необходимых библиотек...
2025-06-21 13:55:48,126 INFO [check_dependencies]: Все необходимые библиотеки присутствуют.
2025-06-21 13:55:48,127 INFO [initialize_database]: Начата инициализация базы данных...
2025-06-21 13:55:48,127 INFO [sync_db_with_fixture]: Синхронизация базы данных с фикстурой...
2025-06-21 13:55:48,152 INFO [sync_db_with_fixture]: Соединение с базой данных установлено
2025-06-21 13:55:48,152 INFO [load_fixture_data]: Загрузка даных из фикстуры: /material.json
2025-06-21 13:55:48,153 ERROR [load_fixture_data]: Файл фикстуры не найден: /material.json
2025-06-21 13:55:48,153 ERROR [sync_db_with_fixture]: Неизвестная ошибка при синхронизации: [Errno 2] No such file or directory: '/material.json'
2025-06-21 13:55:48,153 ERROR [initialize_database]: Ошибка инициализации БД: [Errno 2] No such file or directory: '/material.json'
Traceback (most recent call last):
  File "C:\PetProject\Calculations\ventilated_fasade\main.py", line 68, in initialize_database
    sync_db_with_fixture(FIXTURE_PATH, DB_URL)
  File "C:\PetProject\Calculations\ventilated_fasade\data\sync.py", line 39, in sync_db_with_fixture
    data = load_fixture_data(fixture_path)
  File "C:\PetProject\Calculations\ventilated_fasade\data\sync.py", line 16, in load_fixture_data
    with open(path, 'r', encoding='utf-8') as file:
FileNotFoundError: [Errno 2] No such file or directory: '/material.json'
2025-06-21 13:55:49,639 WARNING [main]: Завершение работы из-за ошибки инициализации БД.
2025-06-21 13:56:00,162 INFO [main]: Запуск калькулятора теплоизоляции НФС v0.1...
2025-06-21 13:56:00,163 INFO [check_dependencies]: Начата проверка наличия необходимых библиотек...
2025-06-21 13:56:00,163 INFO [check_dependencies]: Все необходимые библиотеки присутствуют.
2025-06-21 13:56:00,163 INFO [initialize_database]: Начата инициализация базы данных...
2025-06-21 13:56:00,164 INFO [sync_db_with_fixture]: Синхронизация базы данных с фикстурой...
2025-06-21 13:56:00,190 INFO [sync_db_with_fixture]: Соединение с базой данных установлено
2025-06-21 13:56:00,190 INFO [load_fixture_data]: Загрузка даных из фикстуры: material.json
2025-06-21 13:56:00,190 ERROR [load_fixture_data]: Файл фикстуры не найден: material.json
2025-06-21 13:56:00,191 ERROR [sync_db_with_fixture]: Неизвестная ошибка при синхронизации: [Errno 2] No such file or directory: 'material.json'
2025-06-21 13:56:00,191 ERROR [initialize_database]: Ошибка инициализации БД: [Errno 2] No such file or directory: 'material.json'
Traceback (most recent call last):
  File "C:\PetProject\Calculations\ventilated_fasade\main.py", line 68, in initialize_database
    sync_db_with_fixture(FIXTURE_PATH, DB_URL)
  File "C:\PetProject\Calculations\ventilated_fasade\data\sync.py", line 39, in sync_db_with_fixture
    data = load_fixture_data(fixture_path)
  File "C:\PetProject\Calculations\ventilated_fasade\data\sync.py", line 16, in load_fixture_data
    with open(path, 'r', encoding='utf-8') as file:
FileNotFoundError: [Errno 2] No such file or directory: 'material.json'
2025-06-21 13:56:01,132 WARNING [main]: Завершение работы из-за ошибки инициализации БД.
2025-06-21 13:56:13,211 INFO [main]: Запуск калькулятора теплоизоляции НФС v0.1...
2025-06-21 13:56:13,211 INFO [check_dependencies]: Начата проверка наличия необходимых библиотек...
2025-06-21 13:56:13,212 INFO [check_dependencies]: Все необходимые библиотеки присутствуют.
2025-06-21 13:56:13,212 INFO [initialize_database]: Начата инициализация базы данных...
2025-06-21 13:56:13,212 INFO [sync_db_with_fixture]: Синхронизация базы данных с фикстурой...
2025-06-21 13:56:13,238 INFO [sync_db_with_fixture]: Соединение с базой данных установлено
2025-06-21 13:56:13,238 INFO [load_fixture_data]: Загрузка даных из фикстуры: assets/material.json
2025-06-21 13:56:13,238 ERROR [load_fixture_data]: Файл фикстуры не найден: assets/material.json
2025-06-21 13:56:13,238 ERROR [sync_db_with_fixture]: Неизвестная ошибка при синхронизации: [Errno 2] No such file or directory: 'assets/material.json'
2025-06-21 13:56:13,239 ERROR [initialize_database]: Ошибка инициализации БД: [Errno 2] No such file or directory: 'assets/material.json'
Traceback (most recent call last):
  File "C:\PetProject\Calculations\ventilated_fasade\main.py", line 68, in initialize_database
    sync_db_with_fixture(FIXTURE_PATH, DB_URL)
  File "C:\PetProject\Calculations\ventilated_fasade\data\sync.py", line 39, in sync_db_with_fixture
    data = load_fixture_data(fixture_path)
  File "C:\PetProject\Calculations\ventilated_fasade\data\sync.py", line 16, in load_fixture_data
    with open(path, 'r', encoding='utf-8') as file:
FileNotFoundError: [Errno 2] No such file or directory: 'assets/material.json'
2025-06-21 13:56:16,361 WARNING [main]: Завершение работы из-за ошибки инициализации БД.
